### PR TITLE
Add ability to change drawables programmatically

### PR DESCRIPTION
**Pull request by turing-tech**
Presently the library is not able to accommodate situations where the drawable is variable depending on program conditions. This PR changes this.

### CODE REVIEW

File: library/src/main/java/com/tolstykh/textviewrichdrawable/DrawableEnriched.java [hunk 1]
```diff
@@ -17,4 +19,13 @@
      * @return the width in pixels or {@value UNDEFINED} if undefined.
      */
     int getCompoundDrawableWidth();
+
+    void setDrawableStartVectorId(@DrawableRes int id);
+
+    void setDrawableEndVectorId(@DrawableRes int id);
+
+    void setDrawablTopVectorId(@DrawableRes int id);
```
Comment (a-tolstykh) [inline library/src/main/java/com/tolstykh/textviewrichdrawable/DrawableEnriched.java]: There is a typo in the method name: setDrawablTopVectorId -> setDrawableTopVectorId ('e' is missing in the end of drawable)

File: library/src/main/java/com/tolstykh/textviewrichdrawable/EditTextRichDrawable.java [hunk 2]
```diff
@@ -45,4 +46,25 @@ public int getCompoundDrawableHeight() {
     public int getCompoundDrawableWidth() {
         return mRichDrawableHelper.getCompoundDrawableWidth();
     }
+
+    @Override
+    public void setDrawableStartVectorId(@DrawableRes int id) {
+        mRichDrawableHelper.setDrawableStartVectorId(id);
```
Comment (a-tolstykh) [inline library/src/main/java/com/tolstykh/textviewrichdrawable/EditTextRichDrawable.java]: You forgot to call `mRichDrawableHelper.apply(this);` inside each setter in this class

**Comment by turing-tech** [comment]
I can't approve the changes myself, but if you want to go ahead!

**Comment by a-tolstykh** [comment]
Your changes are already live in 0.3.2!

**Pull request merged**